<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="labelFile1">
                <property name="text">
                 <string>Файл 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditFile1"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonFile1">
                <property name="text">
                 <string>Обзор...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="labelFile2">
                <property name="text">
                 <string>Файл 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditFile2"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonFile2">
                <property name="text">
                 <string>Обзор...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelFile3">
                <property name="text">
                 <string>Файл 3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditFile3"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonFile3">
                <property name="text">
                 <string>Обзор...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Размер буффера</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStart">
                <property name="text">
                 <string>Старт</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTextEdit" name="textEditRes1"/>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditRes2"/>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditRes3"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelBuf">
          <property name="text">
           <string>Буффер</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
